### DISCUSSION TITLE
Not a Bug | Add Discussions to Repo & Status Wiki Sidebar

### DISCUSSION THREAD

**Discussion by lefvv** [editorconfig/editorconfig#507] [General]
Hi:

This is not a bug, but it is filed here per [documentation](https://docs.github.com/en/discussions/quickstart). 

Questions:

1) Any opinions for adding a ['Discussions']((https://docs.github.com/en/discussions/quickstart)) section to the repo? 

For example, I would consider my questions as a better fit for a discussion page. Additionally, it may help shift some issue tickets to a more apt category.

2) Any opinions for adding a Status section to the wiki sidebar? 

In particular, I was wondering what the project status currently is (deprecated, maintenance mode, requesting maintainers, etc). I would be happy to contribute but am unsure if there are any members available to review my PRs.

**Comment by xuhdev** [comment]
1. Good suggestion! I've turned on the discussion tab. When the project started, there was no discussion tab on GitHub.

2. The project has always been maintained so far. However, each plugin has its own maintainer. Some may be active and some may be inactive. There is no single answer to the status of the project.

**Comment by lefvv** [comment]
Appreciate the response :)
